src/main/java/com/taf/business/pages/HomePage.java

```diff
@@ -0,0 +1,22 @@
+package com.taf.business.pages;
+
+import com.taf.core.ConfigManager;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class HomePage {
+
+    private static final String PROJECT_NAME = "//div[@class='projectSelector__project-selector--FXbsR projectSelector__shown--YcXp5']//span[contains(@title, '%s')]";
+    private static final String DASHBOARD_WIDGET = "//div[@class = 'projectSelector__project-selector--FXbsR']";
+
+
+    public void chooseProjectName(){
+        String userLogin = (ConfigManager.getInstance().getConfigDTO().getLogin()).toLowerCase();
```

Comment:
would be great to have some method that return config ConfigManager.getInstance().getConfigDTO()
and use it in the LoginPage.java(line16)

Reply:
Thanks, done.